### PR TITLE
Student team view and rate feature : Interface for the team view (teammates and rate buttons)

### LINKED ISSUE #65: Task.27 AT.02 - Create Acceptance Test for US.02
US.02 states that teachers should be able to login to the webpage.

Acceptance tests for this feature include, but may not be limited to, tests that verify:
 - The database rejects a login attempt with an invalid username
 - The database rejects a login attempt with an invalid password
 - When login data is valid, the database returns a non-null (i.e. non-falsy) object
 - When login data is valid, the database returns the correct teacher object, given a hard-coded teacher object with known username and password fields

### PR DESCRIPTION
**Pull request by JenLys**
image

This feature provides the user with the following:

* an interface displaying the list of teammates whenever a course's "View my Team" button is clicked on
* a rate button is available next to each teammate

Some slight UI enhancements have also been made to the overall StudentView.
Rendering + implementation of a modal for MyTeam.jsx

N.B. I have yet again encountered some more issues with npm run dev

### CODE REVIEW

File: client/src/components/student/MyTeam.jsx
Comment (Vinuyans) [inline client/src/components/student/MyTeam.jsx]: do not merge hardcoded values. these should be removed before merging

File: client/src/components/student/StudentView.jsx
Comment (Vinuyans) [inline client/src/components/student/StudentView.jsx]: This should not be a Modal

File: client/src/components/student/MyTeam.jsx
Comment (JenLys) [reply client/src/components/student/MyTeam.jsx]: got it

File: client/src/components/student/StudentView.jsx
Comment (JenLys) [reply client/src/components/student/StudentView.jsx]: When a course gets clicked it should render the interface from MyTeam.jsx. I understood that I had to implement it as a modal instead of creating a new path for it

**Review by Vinuyans — APPROVED** [review]

**Pull request merged**
